OOP/assignment2/SurveyApplication.java

```diff
@@ -0,0 +1,204 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class SurveyApplication {
+	private static ArrayList<Question> questionList = new ArrayList<Question>();
+	private static ArrayList<Particpant> participantList = new ArrayList<Particpant>();
+
+	public static void main(String args[]) {
+		questionList.add(new Question("Text", "Give your brief feedback"));
+		questionList.add(new Question("Multiple Choice",
+				"how do you rate our company?\n1-excillent 2-good 3-bad"));
+		questionList
+				.add(new Question(
+						"Multiple Selected",
+						"what are the qualities of our company\nfreedom/healthy environment/low work load/good boses"));
+		int good = 0;
+		int excillent = 0;
+		int bad = 0;
+		String personName = "";
+		Scanner scanner = new Scanner(System.in);
+		while (true) {
+
+			System.out.println("******Starting the Survry**********");
+			System.out.println("Enter the name of person ?");
+			personName = scanner.nextLine();
+			System.out.println("Starting of questions");
+			Question question[] = new Question[3];
+			int i = 0;
+
+			for (Question question1 : questionList) {
+
+				String answer = "";
+				System.out.println(question1.getQuestion());
+				if (question1.getType().equals("Text")) {
+					answer = scanner.next();
+				} else if (question1.getType().equals("Multiple Choice")) {
+					int type = scanner.nextInt();
+					if (type == 1) {
```

Comment:
ACTION: Since the options may vary from question to question why to check for only 3 options!